### ISSUE TITLE
apoc.create.virtual.fromNode should wrap the existing node while keeping it's node-id and elementId but it doesn't

### ISSUE THREAD

**Issue by jexp** [neo4j/apoc#570]
e.g. `apoc.create.virtual.fromNode(movie, ["title","released","url"])` 

returns virtual nodes with negative ids but it shouldn't in this case.

Seems like an implementation bug that somehow crept in.

Need it for excluding node embeddings and large text properties for returning in neo4j tools

**Comment by Lojjs** [comment]
@jexp Do you want us in Cypher surface to look into this? In that case could you provide us with a setup so we can reproduce?

**Comment by Lojjs** [comment]
@gem-neo4j did some digging for me and it seems like this was implemented with positive node ids earlier but changed because of a bug where changes to the virtual node were updating the real one: https://github.com/neo4j-contrib/neo4j-apoc-procedures/issues/2252

**Comment by jexp** [comment]
Yeah but that was intentional that it is a wrapper of the real node so all operations on it should pass through. 
Perhaps the naming with "virtual" was not that good it should have been "wrapped".

**Comment by gem-neo4j** [comment]
Cool as it was changed due to a bug fix, I have instead updated so that a config value can be optionally passed in with the key: wrapNodeIds which will give you this behaviour :) https://github.com/neo4j/apoc/pull/712
